app.js

```diff
@@ -48,6 +48,8 @@ app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(fileUpload());
 
+const userPassword = "helloworld";
```

Comment:
### <sub><img src="https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-medium.svg" width="20"/></sub>&nbsp; Use of Hardcoded Credentials

Do not hardcode passwords in code. Found hardcoded password used in userPassword.

<h6>
<a href="https://cwe.mitre.org/data/definitions/798.html">CWE-798</a> | <a href="https://cwe.mitre.org/data/definitions/259.html">CWE-259</a> | <a href="https://docs.snyk.io/manage-risk/prioritize-issues-for-fixing/priority-score">Priority score 562</a> | Line 51</h6>